src/table/settings/driver_settings.ini

```diff
@@ -0,0 +1,148 @@
+; This file is part of OpenTTD.
+; OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+; OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+; See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+;
+
+; Driver settings as stored in the main configuration file ("openttd.cfg").
```

Comment:
The actual driver settings are in misc_settings, and these settings are in my opinion not really driver related. It's currently the sound equivalent of gui.

Reply:
I am open for other suggestions. But neither `music`, nor `sound` covers it, and making 2 tables out of this feels wrong. That is why I came to `driver`, but if you have a better idea, I am all ears.

Reply:
It really depends if you want `driver` to be taken like the rest of the source code uses it or not. Right now, all settings are UI settings that don't relate to anything the source otherwise calls driver.

Our terminology is probably not very common though :)

Reply:
@michicc mentioned on IRC that the 20 year dated name for sound/music is `multimedia`, and I like that name. At least solved this problem; we can always revisit where things go etc at a later date :) This is just a start after all ;)